cypress/integration/specialFeatures/utilities/scriptSwitchingJourneyActions/index.js

```diff
@@ -0,0 +1,41 @@
+const clickFirstLink = () => {
+  cy.get('a[class^="Link"]').first().click();
+};
+
+const clickFirstMapLink = () => {
+  cy.get('div[class^="StyledMediaIndicator"]').then($styledMediaIndicators => {
+    if ($styledMediaIndicators.length > 0) {
+      cy.get('div[class^="StyledMediaIndicator"]')
+        .first()
+        .parentsUntil('li[class^="StoryPromoLi"]')
+        .within(() => {
+          clickFirstLink();
+        });
+    } else {
+      // If a MAP item isn't found on the home page, click the first promo item.
+      clickFirstLink();
+    }
+  });
+};
+
+export const clickScriptSwitcher = variant => {
+  cy.get(`a[data-variant="${variant}"]`).click();
+};
+
+export const clickHomePageLink = product => {
```

Comment:
Do we want to rename this to service or service name?

Reply:
Nice spot! Yes!